client/x/evmstaking/keeper/withdrawal_queue.go

```diff
@@ -17,12 +17,10 @@ func (k Keeper) AddWithdrawalToQueue(ctx context.Context, withdrawal types.Withd
 	return k.WithdrawalQueue.Enqueue(ctx, withdrawal)
 }
 
-func (k Keeper) DequeueEligibleWithdrawals(ctx context.Context) (withdrawals etypes.Withdrawals, err error) {
-	maxDequeue, err := k.MaxWithdrawalPerBlock(ctx)
-	if err != nil {
-		return nil, err
+func (k Keeper) DequeueEligibleWithdrawals(ctx context.Context, maxDequeue int) (withdrawals etypes.Withdrawals, err error) {
```

Comment:
We can simply change like this and no need to check if the value is negative. maxDequeue is a param of max withdrawal per block of evmengine module. 

Also, in the below for loop in this function, no need to wrap `maxDequeue` with `uint64` I think.
```suggestion
func (k Keeper) DequeueEligibleWithdrawals(ctx context.Context, maxDequeue uint32) (withdrawals etypes.Withdrawals, err error) {
```